book/07-extending-learners.qmd

```diff
@@ -102,166 +108,190 @@ For our example, the resulting script looks like this:
 #' @author RaphaelS1
 #' @name mlr_learners_classif.rpart
 #'
-#' @template class_learner
+#' @description
+#' <FIXME> BRIEF DESCRIPTION OF THE LEARNER.
+#' Calls [rpart::rpart()] from <FIXME> (CRAN VS NO CRAN): \CRANpkg{rpart} | 'rpart'.
+#'
+#' @section Custom mlr3 defaults:
+#' <FIXME> DEVIATIONS FROM UPSTREAM PARAMETERS. DELETE IF NOT APPLICABLE.
+#'
+#' @section Custom mlr3 defaults:
+#' <FIXME> DEVIATIONS FROM UPSTREAM DEFAULTS. DELETE IF NOT APPLICABLE.
```

Comment:
`<FIXME>` is ok, many editors highlight fixmes when using `FIXME:` (including the colon).

Reply:
Yes would be better, but will keep it like that for now

Reply:
ok I changed it